data/deep missions.txt

```diff
@@ -273,29 +274,32 @@ mission "Deep: Kraz Convoy"
 	on complete
 		"deep convoy" ++
 		payment 1260000
-		dialog `When you return to <planet> with the freighters, you are greeted by a Deep Security officer who hands you <payment> and thanks you for accepting the job on such a short notice. "We'll make sure to contact you in the future should we need your help."
+		dialog `When you return to <planet> with the freighters, you are greeted by the same Deep Security officer that offered you the job. He hands you <payment>. "Thank you for accepting this job on such a short notice, Captain. I should introduce myself. I am Lieutenant Paris, and I will make sure to contact you in the future should the Deep require your help."`
```

Comment:
I might turn this dialog into a conversation so that I can branch whether or not you've met Paris in Deep: Mystery Cubes 0, thus still allowing this mission to be done if you didn't do it before meeting Paris.